apps/src/aiTutor/chatApi.ts

```diff
@@ -79,24 +78,21 @@ export async function getChatCompletionMessage(
 
   // For now, response will be null if there was an error.
   if (!response) {
-    return {status: Status.ERROR, id: userMessageId};
+    return {status: Status.ERROR};
```

Comment:
Removed ids here because we no longer assign them before we `getChatCompletionMessage`.